lib/tests/2wp.js

```diff
@@ -731,6 +739,44 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should reject and refund a pegout with value exactly below minimum', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+      const initialBtcRecipientAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      const initialRskSenderBalanceInWeisBN = await rskTxHelper.getBalance(senderRecipientInfo.rskRecipientRskAddressInfo.address);
+      // Value exactly below minimum
+      const pegoutValueInRbtc = parseFloat((MINIMUM_PEGOUT_AMOUNT_IN_RBTC - 0.0001).toFixed(8));
```

Comment:
I feel like this could be a lot easier if we stored the min pegout value in satoshis or weis. Less parsing and decimal handling